.github/scripts/publish_post_check.sh

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
Let's remove this along with `stage.yml` and `publish.yml` from the PR. They are no longer in the dev branch.

Reply:
Done.